loops-arrays-objects.js

```diff
@@ -31,6 +33,30 @@ const scores = [
 // A
 // D
 // ==========================================
+for (let i = 0; i < scores.length; i++) {
+    switch (true) {
+        case scores[i].score<60:
+            console.log("F")
+            scores[i].grade="F"
+            break;
+        case scores[i].score<70:
+            console.log("D")
+            scores[i].grade="D"
+            break;
+        case scores[i].score<80:
+            console.log("C")
+            scores[i].grade="C"
+            break;
+        case scores[i].score<90:
+            console.log("B")
+            scores[i].grade="B"
+            break;
+        case scores[i].score<100:
+            console.log("A")
+            scores[i].grade="A"
+            break;
+    }
+}
 
 
```

Comment:
Ziet prima uit, echter zie ik dat je beide opdrachten in een hebt uitgewerkt wat goed uitziet, netjes gedaan, duidelijke code goed te lezen. 
Wanneer iemand een score van 100 heeft behaald, wordt er geen Letter "A", toegevoegd aan je array onder "grade". Aan je laatste case scores[i].score<"="100 de is teken toevoegen .  Verder heb je geen default in je switch? console.log(scores[i]) vergeten?